ncat/ncat_main.c

```diff
@@ -618,6 +626,7 @@ int main(int argc, char *argv[])
 "  -w, --wait <time>          Connect timeout\n"
 "  -z                         Zero-I/O mode, report connection status only\n"
 "      --append-output        Append rather than clobber specified output files\n"
+"      --use-delimiter        Breaks the data till delimiter before broadcasting data; expects ascii value of character (Max range: 255)\n"
```

Comment:
The option name should be `--delimiter`. `--use-delimiter` sounds more like an on/off flag, and this requires an argument. Be sure to note the argument in the help text as `<byte value>`. Description should not mention "broadcast" because that means something different to the user ("send to a broadcast IP address") than it does in our code ("write to all the appropriate file descriptors"). Value is not "ascii" but the decimal byte value 0-255.